### DISCUSSION TITLE
0x8002802B after outlook update and how I resolved the issue.

### DISCUSSION THREAD

**Discussion by Billbarasch** [phw198/OutlookGoogleCalendarSync#1915] [Q&A] [answered]
Hi all, 

Windows 11 all system updates. 

I recently updated my outlook via Windows update. OGCS started giving 0x8002802B messages on startup of the appliaton. 
I attempted repair of outlook and this did not help. I used a restore point via system restore from before the update and re-installed OGCS. I had uninstalled as part of my troubleshooting efforts. 
After the system restore and reinstall OGCS is working again 
I am not using new outlook but the binaries of outlook were updated during the windows update that included Microsoft Products not just operating system. 
My outlook binary is now version 16.0.17726.20160. I did not check what the binary version was before restoring. 
I wanted to share in case this happens to anyone else.

**Comment by phw198** [comment]
Thanks @Billbarasch, I've added a [note to the wiki](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs---COM-Errors/_edit#0x8002802b---type_e_elementnotfound). 

If you have reverted your system to a restore point though, have you updated Office again since? If so, did you not encounter the same issue with OGCS afterwards?

**Reply by Billbarasch** [reply, marked as answer]
I do not intend to update outlook again. OGCS is very important to me and I don't want to lose the ability to run it. 
I am concerned that something in the updated binaries will again break OGCS. So will stay static with outlook versions. 
I hope the OGCS version that is not dependent on Outlook to be running will eliminate the issue. 
Thank you

**Reply by phw198** [reply]
Yes, it's mind boggling how bad Microsoft is at upgrading its own products without breaking them...so if you can avoid updating Office, all the better 🙂
